src/components/ui/socialMedia.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import FacebookIcon from '@mui/icons-material/Facebook';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import InstagramIcon from '@mui/icons-material/Instagram';
+
+const SocialMediaIcons: React.FC = () => {
+  return (
+    <div className="flex justify-center space-x-6">
+      {/* Facebook Icon */}
+      <a
+        href="https://facebook.com"
+        target="_blank"
+        rel="noopener noreferrer"
+        className="text-gray-400 hover:text-white transition-colors duration-200"
+      >
+        <FacebookIcon/>
+      </a>
```

Comment:
These icons need to have a circle background. See

![Screenshot 2024-12-10 at 7 09 42 AM](https://github.com/user-attachments/assets/6bd8db5f-e18d-479e-b73f-2cb5b36f5d99)